go.mod

```diff
@@ -21,24 +21,110 @@ require (
 	golang.org/x/mod v0.4.2
 	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f
 	gopkg.in/gcfg.v1 v1.2.3
-	gopkg.in/warnings.v0 v0.1.2 // indirect
-	k8s.io/api v0.22.2
-	k8s.io/apiextensions-apiserver v0.22.2
-	k8s.io/apimachinery v0.22.2
-	k8s.io/client-go v0.22.2
-	k8s.io/cluster-bootstrap v0.22.2
-	k8s.io/component-base v0.22.2
-	k8s.io/klog v1.0.0
-	k8s.io/klog/v2 v2.9.0
+	k8s.io/api v0.23.0
+	k8s.io/apiextensions-apiserver v0.23.0
+	k8s.io/apimachinery v0.23.0
+	k8s.io/client-go v0.23.0
+	k8s.io/cluster-bootstrap v0.23.0
+	k8s.io/component-base v0.23.0
+	k8s.io/klog/v2 v2.30.0
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
-	sigs.k8s.io/cluster-api v1.0.4
-	sigs.k8s.io/cluster-api/test v1.0.4
-	sigs.k8s.io/controller-runtime v0.10.3
+	sigs.k8s.io/cluster-api v1.1.0
+	sigs.k8s.io/cluster-api/test v1.1.0
+	sigs.k8s.io/controller-runtime v0.11.0
 	sigs.k8s.io/kind v0.11.1
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
```

Comment:
I'm not sure exactly why these were added now, but seems like we have to commit this change anyway.

Reply:
This is go.mod format for Go 1.17, so now direct and indirect dependencies are separate for better readability.